src/app/components/mno-local-products/mno-local-product-listing.coffee

```diff
@@ -33,7 +33,7 @@ angular.module 'mnoEnterpriseAngular'
       # Post-Initialization
       #====================================
       $scope.$watch MnoeOrganizations.getSelectedId, (val) ->
-        vm.initialize() if val?
+        vm.initialize()
```

Comment:
@adamaziz15 any idea why the `if val?` was here? Anything to do with `$watch`?
I've seen this pattern a couple times

Reply:
@adamaziz15  I have tested this scenario locally multiple times, so I'm sure it doesn't. I'll test in UAT for confirmation too.